src/consumer/exceptions/missing-dependencies.js

```diff
@@ -1,9 +1,17 @@
 /** @flow */
-export default class MissingDependencies extends Error {
+import AbstractError from '../../error/abstract-error';
+
+export default class MissingDependencies extends AbstractError {
   components: Object;
 
-  constructor(components) {
+  constructor(components: Object) {
     super();
     this.components = components;
   }
+
+  makeAnonymous() {
+    const clone = this.clone();
+    clone.components = this.toHash(JSON.stringify(clone.components));
```

Comment:
why do you need the json.stringify here?
We won't be able to see anything useful if it's stringify